### PR TITLE
Ruff toml begone part 2

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: test/common/testlib.py [hunk 1]
```diff
@@ -1273,13 +1273,13 @@ def failureHandler(exc_info, python_311=False):
                 (err_case, exc_info) = self.errors[-1]
                 if exc_info:
                     assert err_case == test_case
-                    setattr(test_case, "_ran_debug", True)
+                    test_case._ran_debug = True
```
Comment (allisonkarlitskaya) [inline test/common/testlib.py:1276]: This seems like the sort of thing that might upset vulture...

**Review by allisonkarlitskaya — APPROVED** [review]
Looks fine to me if all the tests pass, but this isn't my code to review, really...

One thing I might change: the first two commits do essentially the same thing, in two different subdirectories, but the commit messages don't make that clear.

File: test/common/testlib.py (hunk 1 above)
Comment (jelly) [reply test/common/testlib.py:1276]: Ah, didn't have vulture installed, but now  I do! Can we not replace that with `ruff` soon :)

Comment (jelly) [reply test/common/testlib.py:1276]:
```
not ok 8 /static-code/test-pyvulture
# test/common/testlib.py:2253: unused attribute '_testlib__nondestructive' (60% confidence)
# test/common/testlib.py:2264: unused attribute '_testlib__no_retry_when_changed' (60% confidence)
# test/common/testlib.py:2275: unused attribute '_testlib__todo' (60% confidence)
# test/common/testlib.py:2318: unused attribute '_testlib__timeout' (60% confidence)
```
Meh

File: test/common/storagelib.py [hunk 2]
```diff
@@ -438,10 +438,10 @@ def udisks_objects(self):
                 "GetManagedObjects", "", [])))""")]))
 
     def configuration_field(self, dev, tab, field):
-        all = self.udisks_objects()
-        for path in all:
-            if "org.freedesktop.UDisks2.Block" in all[path]:
-                iface = all[path]["org.freedesktop.UDisks2.Block"]
+        managerObjects = self.udisks_objects()
+        for path in managerObjects:
+            if "org.freedesktop.UDisks2.Block" in managerObjects[path]:
```
Comment (martinpitt) [inline test/common/storagelib.py:443]: This is rather bad style anyway -- should be `for path, obj in self.udisks_objects().items()`, or possibly even `for obj in self.udisks_objects.values()` if `path` isn't used anywhere else.

**Review by martinpitt — APPROVED** [review]
Thanks! There is this [pybridge failure](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18921-20230612-165453-b6864df3-fedora-38-pybridge/log.html#342-2) in TestJournal.testRebootAndTime. I've seen it somewhere else yesterday already, so retrying once. It could also be that this slipped into main somehow.

**Pull request merged**